inspirehep/modules/records/tasks.py

```diff
@@ -204,3 +210,38 @@ def actions():
         'success': success,
         'failures': [failure for failure in failures or []],
     }
+
+
+@shared_task(ignore_result=False, bind=True)
+def index_new_cited_records(self, pid_type, pid_value, db_version):
+    record = get_db_record(pid_type, pid_value)
+
+    if record.model.version_id < db_version:
+        backoff = 2 ** (self.request.retries + 1)
+        eta = datetime.datetime.now() + datetime.timedelta(seconds=backoff)
+        logger.warn(
+            'Record {} not yet updated to version {} on DB'.format((pid_type, pid_value), db_version)
+        )
+        raise self.retry(eta=eta)
+
+    pids = record.get_modified_references()
+
+    if not pids:
+        logger.info('No references change for record {}'.format((pid_type, pid_value)))
+        return
+
+    uuids = [
+        str(pid.object_uuid) for pid in
+        db.session.query(PersistentIdentifier.object_uuid).filter(
+            PersistentIdentifier.object_type == 'rec',
+            tuple_(PersistentIdentifier.pid_type, PersistentIdentifier.pid_value).in_(pids)
+        )
+    ]
+
+    if uuids:
+        return batch_reindex.apply_async(
```

Comment:
can you really schedule a task from another task? I remember vaguely that celery was complaining about this kind of things during the 3->4 migration, but don't remember the specifics.

Reply:
Leftover from the previous pr :+1: